css/aboutpage.css

```diff
@@ -0,0 +1,334 @@
+  line-height: 28px;
+}
+
+.logo-section-text {
+  margin-top: 10px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 21px;
+  color: #838383;
+  text-align: center;
+}
+
+.logo-section-img-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 90%;
+}
+
+.logo-section-img {
+  width: 90%;
+}
+
+.past-rock {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin-top: 35px;
+}
+
+.past-img1-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  text-align: center;
+}
+
+.past-img1-container {
+  width: 334px;
+  height: 182px;
+  background: url("../Assets/mobile-img2.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  margin-bottom: 25px;
+  color: #fff;
+}
+
+.past-img1-overlay {
+  width: 334px;
+  height: 182px;
+  background: rgba(230, 61, 9, 0.6);
+}
+
+.past-img2-container {
+  width: 334px;
+  height: 182px;
+  background: url("../Assets/mobile-img3.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.past-img2-overlay {
+  width: 334px;
+  height: 182px;
+  background: rgba(230, 61, 9, 0.6);
+}
+
+.past-img3-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  text-align: center;
+  color: #fff;
+}
+
+.sponsor-title {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin-bottom: 0;
+  font-weight: 500;
+  padding: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+.sponser-about {
+  background-color: #414246;
+  color: #fff;
+  padding: 1rem 0;
+  position: relative;
+  overflow: hidden;
+  margin-top: 35px;
+}
+
+.past-text {
+  margin-top: 10px;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 12px;
+  line-height: 21px;
+  color: #838383;
+  text-align: center;
+  margin-bottom: 35px;
+}
+
+.footer-about {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 86px;
+}
+
+.past-imgs-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.footer-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #fff;
+}
+
+.footer-address {
+  color: #3b3e45;
+}
+
+.footer-para {
+  font-weight: 700;
+  font-size: 13px;
+  line-height: 15px;
+}
+
+.footer-para-desktop {
+  display: none;
+}
+
+.footer-logo-img-about {
+  width: 85px;
+}
+
+.mob-menu2 {
+  width: 100%;
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background: rgba(230, 61, 9, 0.6);
+  color: #fff;
+  display: none;
+  justify-content: space-between;
+  padding: 40px;
+  backdrop-filter: blur(10px);
+}
+
+.closeBtn2 {
+  font-size: 35px;
+  width: 22px;
+  height: 22px;
+}
+
+@media screen and (min-width: 768px) {
+  .about-container {
+    padding: 80px 20px;
+  }
+
+  .logo-section-img {
+    /* width: 90%; */
```

Comment:
- [ ] Please remove all comment pieces of code in all your documents, this will keep your code clean and professional.